lib/cli.js

```diff
@@ -12,9 +12,14 @@ var meow = require('meow');
 var list = require('cli-list');
 var pkg = require('../package.json');
 var Router = require('./router');
-
 var gens = list(process.argv.slice(2));
 
+/* eslint new-cap: 0, no-extra-parens: 0 */
+var tabtab = new (require('tabtab').Commands.default)({
```

Comment:
Yeah, you should add the babel plugin to make babel compiled code compatible with Node commonjs modules.

https://github.com/59naga/babel-plugin-add-module-exports